### ISSUE TITLE
Reactions menu cannot be opened with applications key or shift+f10

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2762]
**Describe the bug**
The reactions menu can only be opened with the mouse (or by emulating a mouse right click).

**To Reproduce**
1. Focus a message which can be reacted.
2. Press applications key or shift+f10.

**Expected behavior**
The reactions menu should open. (Currently it is not accessible, but that's for another issue).

**Version Info**
 - Unigram: Unigram 8.8 (7407) X64 Direct
 - Windows: Versión 21H2 (compilación de SO 19044.1766)
